pymc_marketing/mmm/utils.py

```diff
@@ -33,3 +36,43 @@ def generate_fourier_modes(
             for func in ("sin", "cos")
         }
     )
+
+
+def michaelis_menten(x, L, k) -> float:
```

Comment:
This is also useful as a dedicated saturation function in mmm.transformers

Reply:
I like this idea, checking Google MMM lightweight they had included several model configurations (Hill, Carryover, Adstock) which you could choose to define Saturation and Lagging.
**Example:**
<img width="854" alt="Screenshot 2023-08-01 at 14 49 09" src="https://github.com/pymc-labs/pymc-marketing/assets/59846724/fb5f6d66-03fb-4657-a7ef-57b557ca005c">

They refer to the following wiki where the saturation mentioned it is very similar to the `Michaelis Menten`. I could imagine we can implement something similar here, in order to be more extensive to how different existing data sets can fit better to different saturation and lagging functions depending on their own conditions.

<img width="964" alt="Screenshot 2023-08-01 at 14 46 00" src="https://github.com/pymc-labs/pymc-marketing/assets/59846724/a66201c6-c04f-4d90-ac81-c385daf85262">